app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,278 @@
+class OrdersController < ApplicationController
```

Comment:
I wrote the below comment on the index action but wanted to say it applied to almost every method in this controller.